gitlab/resource_gitlab_user_test.go

```diff
@@ -96,7 +97,12 @@ func TestAccGitlabUser_password_reset(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckGitlabGroupDestroy,
 		Steps: []resource.TestStep{
-			// Create a user
+			// Test that either password or reset_password is needed
+			{
+				Config:      testAccGitlabUserConfigWrong(rInt),
+				ExpectError: regexp.MustCompile(`\sAt least one of either password or reset_password must be defined`),
```

Comment:
Nitpick but the `\s` made me scratch my head for a minute. I think the capital `A` alone serves the intended purpose with less head scratching. Thoughts?

```suggestion
				ExpectError: regexp.MustCompile("At least one of either password or reset_password must be defined"),
```